scripts/setup.sh

```diff
@@ -18,7 +18,7 @@ RELEASE_COMPILER="4.06.0+flambda"
 
 # Compatibility settings with MacOS.
 if [[ "${MACHTYPE}" = *apple* ]]; then
-  export MACOSX_DEPLOYMENT_TARGET=10.11
+  export MACOSX_DEPLOYMENT_TARGET="$(sw_vers -productVersion | cut -d '.' -f 1,2)"
```

Comment:
This hardcoded value is actually intended, it represents the minimum version of OSX supported by Pyre. Replacing it with the `sw_vers` command would make it dynamic instead, and always equivalent to the current version of OSX (hence, possibly wrong).

Reply:
++, Thanks for catching, will change 👍